app.py

```diff
@@ -1,22 +1,76 @@
+import os
+import re
+import string
 import config
-from os import urandom
-from core.helpers import initialize
+
+
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
+from flask_graphql import GraphQLView
+from flask_graphql_auth import GraphQLAuth
+
+from dvga.helpers import initialize
+from dvga.helpers import level_is_easy
+from dvga.helpers import level_is_hard
+
+from secrets import choice
+
 
 app = Flask(__name__, static_folder="static/")
-app.secret_key = urandom(24)
+app.schema = None
+chrset = string.ascii_lowercase + string.digits
+
+# Weakened deliberately to allow for JWT bruteforce 
+app.secret_key = ''.join([choice(chrset) for _ in range(6)]) 
```

Comment:
I like the idea of including a JWT bruteforce :) in order to improve readability and clarity consider pulling this into a separate PR and updating the README scenarios to include the new vulnerability.

Reply:
I have added multiple vulnerabilities and tried to comment in the code in order to make them visible. Would you like me to do a separate PR for every vulnerability I added or plan to add?